python/ray/tests/test_autoscaler_fake_multinode.py

```diff
@@ -100,6 +101,51 @@ def i():
     # __example_end__
 
 
+@pytest.mark.parametrize("autoscaler_v2", [False], ids=["v1"])
+@pytest.mark.parametrize("placement_strategy", ["SPREAD", "STRICT_SPREAD"])
+def test_fake_autoscaler_placement_group_partial_idle(
+    autoscaler_v2, placement_strategy
+):
+
```

Comment:
can you add the comment to explain the case you want to test?

Reply:
👍